framework/src/Volo.Abp.Auditing/Volo/Abp/Auditing/JsonNetAuditSerializer.cs

```diff
@@ -18,10 +20,29 @@ public string Serialize(object obj)
         {
             var options = new JsonSerializerSettings
```

Comment:
Why not using the whole JsonSerializerSettings object instead of only reusing the  contract resolver?